cvmfs/cvmfs_config

```diff
@@ -1833,15 +1833,23 @@ EOF
      # SIP) create directory in root `/`.
      # We decide to move the standard installation directory on
      # `/Users/shared/cvmfs` instead of `/cvmfs`
+     # In order to keep the standard CVMFS mountpoint, we decide to implement
+     # firmlinks from `/cvmfs` to `/Users/shared/cvmfs`
      if sw_vers | grep 10.15; then
-        sudo mkdir -p /Users/shared/cvmfs
-        sudo chown cvmfs:cvmfs /Users/shared/cvmfs
-        sudo chmod 777 /Users/shared/cvmfs
+        mkdir -p /Users/shared/cvmfs
+        chown cvmfs:cvmfs /Users/shared/cvmfs
+        chmod 777 /Users/shared/cvmfs
+        if ! grep -q "cvmfs" /etc/synthetic.conf 2> /dev/null ; then
+          echo -e "# Autogenerated by CernVM-FS, do not delete nor edit" >> /etc/synthetic.conf
```

Comment:
I think we cannot do that.  There might be other parties interested in editing the _file_. We can only set the "auto-generated" marker for our _line_, if that is possible (i.e., if lines can have trailing comments).  Or, we have to wrap the `cvmfs` line between begin and end comment lines.

Reply:
Aw I meant that the line below was autogenerated, not the whole file.

Something like this:

```
# The line below was autogenerated by CernVM-FS, do not delete nor edit it
cvmfs /Users/Shared/cvmfs
```

Reply:
The manual of `syntetich.conf` indicate that only the line that starts with `#` are comment, so we should not use trailing comments.